src/biblebot/triggers.py

```
@@ -0,0 +1,192 @@
+"""Trigger policy module for BibleBot.
+
+Encapsulates all scripture reference detection logic. The main entry point is
+detect_trigger(), which takes message content and bot configuration and returns
+a TriggerMatch when a valid reference is found, or None otherwise.
+"""
+
+import re
+from dataclasses import dataclass
+from enum import Enum
+from typing import Sequence
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)